### DISCUSSION TITLE
can I show the tooltip after scroll end?

### DISCUSSION THREAD

**Discussion by tjx666** [gilbarbara/react-joyride#1039] [General]
See following toggle to next step.

When I click the next:

1. the tooltip move the next quickly and show in top, but there is no enough space for it. Event If I show on bottom, also no enough space
2. after scroll, the is enough space on the bottom

My expect is: don't show the tooltip when scrolling, show it after scroll end

https://github.com/gilbarbara/react-joyride/assets/41773861/621327d4-3919-4dad-8ba3-c23efa7d8af9

**Comment by gilbarbara** [comment]
Hey @tjx666 

No, there isn't an option to change this behavior.

**Comment by gilbarbara** [comment]
Fixed in v3.0.0

Full migration guide: https://react-joyride.com/docs/migration
What's new: https://react-joyride.com/docs/new-in-v3

Thanks
